vue/components/ui/checkbox-platforme/checkbox-platforme.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+    <div class="checkbox-group">
+        <label-platforme class="title" v-bind:text="labelTitle" v-if="labelTitle" />
+        <div class="checkbox-group-choices">
+            <div
+                class="checkbox-choice"
+                v-bind:class="{
+                    checked: values[item.value],
+                    disabled: disabled || item.disabled,
+                    error: error || item.error
+                }"
+                v-for="(item, index) in items"
+                v-bind:key="index"
+                v-on:click="onClick(item)"
+            >
+                <input type="checkbox" class="value" v-bind:id="item.value" />
+                <div class="checkbox-square" />
+                <label class="label" for="item.value">
+                    {{ item.label ? item.label : item.value }}
+                </label>
+            </div>
+        </div>
+        <label-platforme
+            class="footer"
+            v-bind:size="'small'"
+            v-bind:text="labelFooter"
+            v-if="labelFooter"
+        />
+    </div>
+</template>
+
+<style lang="scss" scoped>
+@import "css/variables.scss";
+
+.checkbox-group {
+    display: inline-block;
+}
+
+.checkbox-group > .title {
+    color: $pale-grey;
+    display: block;
+}
+
+.checkbox-choice {
+    display: block;
+    line-height: 13px;
+    padding: 10px 6px 10px 6px;
+    user-select: none;
+}
+
+.checkbox-choice:hover,
+.checkbox-choice:hover ::v-deep label {

```

Comment:
Is this line needed?

Reply:
Updated `.checkbox-choice:hover ::v-deep label` -> `.checkbox-choice:hover label`
But it is still needded to override cursor default:

<img width="407" alt="imagem" src="https://user-images.githubusercontent.com/24736423/70054903-5a43e000-15d0-11ea-90af-243ef94816d4.png">